install.sh

```diff
@@ -19,46 +19,46 @@ touch $HOME/.ssh/known_hosts
 
 echo "================= Installing basic packages ==================="
 apt-get install -y \
-  sudo  \
-  build-essential \
-  curl \
-  gcc \
-  make \
-  openssl \
-  software-properties-common \
-  wget \
-  nano \
-  unzip \
-  openssh-client \
-  libxslt-dev \
-  libxml2-dev \
-  htop \
-  gettext \
-  texinfo
+  sudo=1.8.9p5-1ubuntu1  \
+  build-essential=11.6ubuntu6 \
+  curl=7.35.0-1ubuntu2.10 \
+  gcc=4:4.8.2-1ubuntu6 \
+  make=3.81-8.2ubuntu3 \
+  openssl=1.0.1f-1ubuntu2.22 \
+  software-properties-common=0.92.37.8 \
+  wget=1.15-1ubuntu1.14.04.2 \
+  nano=2.2.6-1ubuntu1 \
+  unzip=6.0-9ubuntu1.5 \
+  openssh-client=1:6.6p1-2ubuntu2.8 \
+  libxslt1-dev=1.1.28-2ubuntu0.1 \
+  libxml2-dev=2.9.1+dfsg1-3ubuntu4.9 \
+  htop=1.0.2-3 \
+  gettext=0.18.3.1-1ubuntu3 \
+  texinfo=5.2.0.dfsg.1-2 \
+  rsync=3.1.0-2ubuntu0.2
 
 echo "================= Installing Python packages ==================="
 apt-get install -y \
-  python-pip \
-  python-software-properties \
-  software-properties-common \
-  python-dev
+  python-pip=1.5.4-1ubuntu4 \
+  python-software-properties=0.92.37.8 \
+  python-dev=2.7.5-5ubuntu3
 
-#update pip version
+# Update pip version
 python -m pip install -U pip
 pip install virtualenv
 
 echo "================= Installing Git ==================="
 add-apt-repository ppa:git-core/ppa -y
 apt-get update
-apt-get install -y git
+apt-get install -y git=1:2.13.0-0ppa1~ubuntu14.04.1
```

Comment:
This changed from `1:2.11.0-2~ppa0~ubuntu14.04.1` as this version is no longer available. This is a minor update, but shouldn't have any critical breaking changes:

```
root@new-image:/# apt-cache madison git         
       git | 1:2.13.0-0ppa1~ubuntu14.04.1 | http://ppa.launchpad.net/git-core/ppa/ubuntu/ trusty/main amd64 Packages
       git | 1:1.9.1-1ubuntu0.5 | http://archive.ubuntu.com/ubuntu/ trusty-updates/main amd64 Packages
       git | 1:1.9.1-1ubuntu0.5 | http://archive.ubuntu.com/ubuntu/ trusty-security/main amd64 Packages
       git |  1:1.9.1-1 | http://archive.ubuntu.com/ubuntu/ trusty/main amd64 Packages
       git |  1:1.9.1-1 | http://archive.ubuntu.com/ubuntu/ trusty/main Sources
       git | 1:1.9.1-1ubuntu0.5 | http://archive.ubuntu.com/ubuntu/ trusty-updates/main Sources
       git | 1:1.9.1-1ubuntu0.5 | http://archive.ubuntu.com/ubuntu/ trusty-security/main Sources
```